Userland/Utilities/groupdel.cpp

```diff
@@ -68,20 +60,22 @@ int main(int argc, char** argv)
     char temp_group[] = "/etc/group.XXXXXX";
 
     auto unlink_temp_files = [&] {
-        if (unlink(temp_group) < 0)
+        if (Core::System::unlink(temp_group).is_error())
             perror("unlink");
     };
 
     ArmedScopeGuard unlink_temp_files_guard = [&] {
         unlink_temp_files();
     };
 
-    auto temp_group_fd = mkstemp(temp_group);
-    if (temp_group_fd == -1) {
+    auto mkstemp_or_error = Core::System::mkstemp(temp_group);
+    if (mkstemp_or_error.is_error()) {
```

Comment:
The whole point of these `TRY()`-able wrappers is that we don't have to manually make an error message and return :)

This, and everything else of the same pattern, should just be `auto temp_group_fd = TRY(Core::System::mkstemp(temp_group));`

Reply:
I thought keeping a more verbose error message would be better (for instance this one has "failed to create temporary group file"). I will change these accordingly.